app/javascript/react/screens/App/Overview/screens/formUtilities/renderField.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Grid } from 'patternfly-react';
+
+export const renderField = ({
```

Comment:
As this is a React component, it should probably be capitalized, and perhaps be a "noun" name instead of a "verb" name. `FormField`?

Reply:
Addressed in d3da630